lib/api_callable.js

```diff
@@ -228,7 +237,13 @@ function retryable(aFunc, retry, otherArgs) {
         }
       });
     }
-    repeat();
+
+    if (maxRetries && deadline) {
+      callback(new Error('Cannot have both totalTimeoutMillis and maxRetries' +
+          'set in backoffSettings.'));
```

Comment:
This would be simpler if it said "Cannot set both totalTimeoutMillis and maxRetries in backoffSettings."